app/src/main/java/com/github/houseorganizer/houseorganizer/CreateHouseholdActivity.java

```diff
@@ -0,0 +1,63 @@
+package com.github.houseorganizer.houseorganizer;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import com.google.firebase.firestore.FirebaseFirestore;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+public class CreateHouseholdActivity extends AppCompatActivity {
+
+    private String Uid;
+    private FirebaseFirestore db;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_create_household);
+
+        db = FirebaseFirestore.getInstance();
+        Uid = getIntent().getStringExtra("Uid");
+    }
+
+    public void submitHouseholdToFirestore(View view){
+        TextView houseHoldNameView = findViewById(R.id.editTextHouseholdName);
+        CharSequence houseHoldName = houseHoldNameView.getText();
+
+        Map<String, Object> houseHold = new HashMap<>();
+        Map<String, Object> residents = new HashMap<>();
+
+        residents.put("0", Uid);
```

Comment:
I know nothing about firebase but is it possible to have a <Integer, Object> map ?

Reply:
It is a string, not an integer ;)